Source/WebInspectorUI/UserInterface/Models/DOMNode.js

```diff
@@ -1321,6 +1321,12 @@ WI.DOMNode = class DOMNode extends WI.Object
         if (this._layoutOverlayShowing)
             this.showLayoutOverlay();
     }
+
+    async getMediaStats()
```

Comment:
I was going to suggest using a property here, since that would be more aligned with the Inspector coding style, but it appears you can't have `async get mediaStats()`. You could still do that by returning a `Promise`. It's really up to you.